server/controllers/lecture-controller.js

```diff
@@ -246,7 +249,7 @@ exports.convertDistanceLecture = async (req, res) => {
     .format("YYYY-MM-DD HH:mm:ss");
   knex
     .select(
-      { name: "name" },
+      //{ name: "name" },
```

Comment:
Usually it's seen as a bad practice to commit files containing commented lines of code.